index.html

```diff
@@ -36,11 +24,11 @@
     <nav>
     </header>
     <section id="welcome-section">
-      <h1>Hey we are Developers</h1>
+      <h1>Hello, I'm a developper</h1>
     </section>
 
     <section id="projects">
-      <h2 class="project-tile">These are some of our project </h2>
+      <h2 class="project-tile">These are some of our projects </h2>
       <div class="projects-grid">
     <a
```

Comment:
Please use flex property to align your header items as it helps with the flexibility of your nav items 😄